tests/UnitTests/Foundation/NSRunLoopTests.mm

```diff
@@ -0,0 +1,48 @@
+//******************************************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#include <TestFramework.h>
+#include <Foundation/Foundation.h>
+
+@interface NSRunLoopTest : NSObject
+- (void)shouldNotBeCalled;
+@end
+
+@implementation NSRunLoopTest
+
+- (void)shouldNotBeCalled {
+    FAIL();
+}
+
+@end
+
+TEST(NSRunLoop, CancelPerformSelector) {
```

Comment:
We haven't yet completed the run-loop migration to use CF.  Any additional testing can be covered there.